qa/qa-captain/bashrc

```diff
@@ -0,0 +1,1116 @@
+        # diff --color -s --suppress-common-lines system-agent-installers.txt upgrade-images.txt
+    ;;
+    esac
+}
+
+# --- check for upgrade images in 
+
+
+# ======================================================================================
+# ================================== ~ GITHUB ~ =======================================
+# ======================================================================================
+
+# --- list issues in the github repos using the gh cli --- 
+ function ghls() {
+	local PRODUCT="${1:-k3s}"
+    local USER="${2:-VestigeJ}"
+    case $PRODUCT in
+    qa) gh issue ls -R rancher/qa-tasks -a "$USER" ;;
+	rke2) gh issue ls -R rancher/rke2 -a "$USER" ;;
+	k3s) gh issue ls -R k3s-io/k3s -a "$USER" ;;
+	na)
+	  printf "K3S-ISSUES---------------------------------------"
+	  gh issue ls -R k3s-io/k3s 
+	  printf "\nRKE2-ISSUES------------------------------------"
+	  gh issue ls -R rancher/rke2
+      printf "\nQA-ISSUES---------------------------------------"
+      gh issue ls -R rancher/qa-tasks
+	  ;;
+	esac
+	}
+
+# --- get github issue comments ---
+ function ghcom() {
+	local PRODUCT="${1:-k3s}"	
+	local ISSUE="${2}"
+    case $PRODUCT in
+	rke2)
+	  gh issue view -c "$ISSUE" -R rancher/rke2
+	  ;;	
+	k3s)
+	  gh issue view -c "$ISSUE" -R k3s-io/k3s
+	  ;;
+    qa)
+      gh issue view -c "$ISSUE" -R rancher/qa-tasks
+      ;;
+	esac
+	}
+
+# --- list issues closed in the last n days for the milestone ---
+function ghclosed() {
+    local PRODUCT="${1:-k3s}"
+    local days="${2:-7}"
+    local date
+    date=$(date -v -"$days"d +%F)
+    local MILESTONE="${3:-v1.25.3+k3s1}"
+    case $PRODUCT in
+    rke2)
+      gh search issues --repo rancher/rke2 --closed \>"$date" --milestone "$MILESTONE"
+      ;;
+    k3s)
+        gh search issues --repo k3s-io/k3s --closed \>"$date" --milestone "$MILESTONE"
+        ;;
+    qa)
+        gh search issues --repo rancher/qa-tasks --closed \>"$date"
+        ;;
+    esac
+    }
+
+# --- list issues in milestone ---
+function ghmile() {
+    local MILESTONE="${1}"
+    local PRODUCT="${2:-k3s}"
+    case $PRODUCT in
+    rke2)
+        gh issue list --milestone "$MILESTONE" -R rancher/rke2
+        ;;
+    k3s)
+        gh issue list --milestone "$MILESTONE" -R k3s-io/k3s
+        ;;
+    esac
+}
+
+# ======================================================================================
+# ================================== ~ ALIASES ~ =======================================
+# ======================================================================================
+#
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/var/lib/rancher/k3s/bin:/var/lib/rancher/rke2/bin:/usr/local/bin/go/bin:/opt/rke2/bin/:${HOME}/.krew/bin
+alias ll="ls -lahr "
```

Comment:
Looks like there are a couple different alias sections. Should these be consolidated into 1?

Reply:
Yes - the alias' from the default Ubuntu bashrc were floating in the top. I've moved many down and removed what I imagine to be useless.